src/query/api/v1/handler/influxdb/rewrite.go

```diff
@@ -0,0 +1,54 @@
+/*
+ * Author: Markus Stenberg <mstenber@aiven.io>
+ *
+ * Copyright (c) 2019 Aiven Oy
+ */
```

Comment:
nit: These headers will have to be replaced with the default headers as appears in all other source in the repo.